const { generateError } = require("../../helpers");

const deleteCategory = async (req, res, next) => {
  try {
    
    
    res.send({
      status: "ok",
      message: "CategorÃ­a eliminada ğŸ”´",
    });
  } catch (err) {
    next(err);
  }
};

module.exports = deleteCategory;
